### DISCUSSION TITLE
Disable all plugins button

### DISCUSSION THREAD

**Discussion by SweaterPuppies** [runelite/runelite#16198] [Ideas]
I'm having issues and it would be nice if there was a disable all plugins button instead of having to individually turn them all off.

**Comment by Alexsuperfly** [comment]
basically duplicate of #10937

**Reply by SweaterPuppies** [reply]
> basically duplicate of #10937

The OP did not use the word plugin.

**Comment by Espionage724** [comment]
I reinstall operating systems and RL frequently. I don't like majority of the default-enabled plugins, and only really use RL for the the GPU plugin while on Linux.

Currently I toggle-off every plugin one-by-one down the list, and this is cumbersome. I would prefer a setting or method to turn off all plugins all at once, and allow me to enable the few I want.

I'd prefer it to be some easy-click button, but I wouldn't mind it being some hidden file or easy text config.
